### ISSUE TITLE
temperature settings "stuck" after using Per Model Settings

### ISSUE THREAD

**Issue by yourhead** [Ultimaker/Cura#2488]
Days ago I created a project test speeds with some new filament. The project had several models. Each model had some "Per Model Settings".

Since this time all gcode produced outputs to the same temperature setting I used for that run:  245°.

I've tried changing everything including temperatures, hiding/showing more settings, re-creating the multi-model project -- but nothing seems to have any affect on the produced gcode.

To be 100% clear: the "Printing Temperature" setting is currently set to 195. But the output gcode continues to be 245.

I'd be happy to provide logs, gcode files, etc if you let me know what you need to investigate further.

**Comment by nallath** [comment]
Could you share the project file with us? That makes it a lot easier to follow what you did exactly.

**Comment by yourhead** [comment]
Sure.
https://www.dropbox.com/s/ztufmngvuh45a3e/Project.zip?dl=0

To be clear:  This behavior exists on ALL projects now. It is not associated with any single project.

So, I've just created a project from scratch with an STL file I had around. I've included the project file (.3mf), the gcode that cura writes out, as well as the profile that i used (curaprofile).

The profile in question was not overridden in the UI. It displays as having a temperature of 195º. If there is some other hidden temperature that supersedes this -- it would probably (especially for temperature) be a good idea to display what the user is going to get **somewhere** pretty obvious.

in other words:  if the answer to this is that i have some hidden property. then this bug report is "that property should never be hidden"

In my case the gcode shows 245
<img width="101" alt="pasted_image_9_27_17__11_34_am" src="https://user-images.githubusercontent.com/98634/30925556-d8d768c4-a377-11e7-978d-7f05470c68c9.png">

**Comment by nallath** [comment]
I've just tried this with master (on a Custom FDM Printer). 
It seems to work just fine; 

![g-codetemperature](https://user-images.githubusercontent.com/3922611/30975354-5d9e309c-a472-11e7-8906-ddf64bf49d1f.png)
![curatemperaturesettings](https://user-images.githubusercontent.com/3922611/30975355-5d9f4fa4-a472-11e7-974c-cba7e1663f43.png)

Could you share me the project file and not just the .3mf file?
